docs/addons/secrets-store.md

```diff
@@ -0,0 +1,62 @@
+# Secrets Store Add-on
+
+The Secrets Store Add-on provisions the [AWS Secrets Manager and Config Provider(ASCP) for Secret Store CSI Driver](https://docs.aws.amazon.com/secretsmanager/latest/userguide/integrating_csi_driver.html) on your EKS cluster. With ASCP, you now have a plugin for the industry-standard Kubernetes Secrets Store [Container Storage Interface (CSI) Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) used for providing secrets to applications operating on Amazon Elastic Kubernetes Service.
+
+With ASCP, you can securely store and manage your secrets in [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager) or [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) and retrieve them through your application workloads running on Kubernetes. You no longer have to write custom code for your applications.
+
+> **Compatibility**: This addon is only compatible with following Cluster Providers.<br/> [EC2 Cluster Provider](./../cluster-providers/ec2-cluster-provider.md)
```

Comment:
Can we add detail on how the customer consumes secrets with this approach? Are they boostrapped in the pod ENV?

Reply:
Yes, working on it.

Reply:
Addressed.

Reply:
Does it imply it is incompatible with Bottlerocket cluster provider? Can you point me to the doc?

Reply:
I don't recall why I added that. It may be a misunderstanding on my part. I will test it with all 3 providers when its ready. Until then removing the Compatibility note.